scripts/arm32_ci_script.sh

```diff
@@ -23,7 +23,9 @@ function usage {
     echo '    --buildConfig=<config>             : The value of config should be either Debug or Release'
     echo '                                         Any other value is not accepted'
     echo 'Optional Arguments:'
+    echo '    --arm                              : Build as arm (default)'
     echo '    --armel                            : Build as armel'
+    echo '    --linuxCodeName=<name>             : Linux code name: trusty, xenial (default) and tizen'
```

Comment:
Why is xenial default here and not trusty (as at https://github.com/dotnet/coreclr/blob/master/cross/build-rootfs.sh#L13)? We should keep consistency for the defaults across the board so that it is simple to follow.